src/user/user-section.css

```
@@ -121,4 +122,19 @@ article.pf-c-card.pf-m-flat.kc-available-idps > div > div > h1 {
 
 .keycloak__user-credentials__reset-form {
   --pf-c-form--m-horizontal__group-label--md--GridColumnWidth: 13rem;
-}
\ No newline at end of file
+}
+
+.pf-c-card__header.kc-form-panel__header{
+  padding-bottom: 0px;
```

You can omit the unit if the value is `0`.
```suggestion
  padding-bottom: 0;
```